packages/vuetify/src/components/VDatePicker/VDatePickerMonth.tsx

```diff
@@ -58,7 +58,10 @@ export const VDatePickerMonth = genericComponent<VDatePickerMonthSlots>()({
   setup (props, { emit, slots }) {
     const daysRef = ref()
 
-    const { daysInMonth, model, weekNumbers } = useCalendar(props)
+    const { daysInMonth, model, weekNumbers } = useCalendar({
+      ...props,
+      displayValue: null,
+    })
```

Comment:
Thoughts on reverting this and instead making displayValue optional in the CalendarProps interface?
![image](https://github.com/user-attachments/assets/73b12a61-8233-4a30-87c0-33f0166d8f80)

Reply:
I agree, it should not be required in `CalendarProps`.